### DISCUSSION TITLE
Home Assistant Green

### DISCUSSION THREAD

**Discussion by Menhir83** [indomus/forum#3126] [Hardware & Firmware]
Salve a tutti, mi sono appena affacciato al mondo Home Assistant e ho una domanda. Ho trovato una community dove spiegano veramente tanto e in modo chiaro tutto quello che cé da sapere. Peró ho trovato una soluzione che non viene contemplata nemmeno in questo forum. Sto parlando di Home Assistant Green. Da ignorante in materia, a paritá di prezzo, sembra una buona soluzione per iniziare. Qualcuno ha esperienze da condividere?
ps: é la mia prima discussione, spero di non aver sbagliato nulla
Grazie in anticipo

**Comment by indomus** [comment]
Hola. Non è una buona soluzione per il semplice fatto che il rapporto qualità/prestazioni/prezzo è pessimo. Sbandierato come la soluzione definitiva, ma molto meglio un mini pc per [tante buone ragioni](https://indomus.it/formazione/domotica-su-home-assistant-mi-conviene-un-nuc-un-raspberry-pi-o-altro/).

**Reply by Menhir83** [reply]
Ho capito. Cercavo una soluzione veloce e semplice perché uno dei miei Laptop si é fritto e l´altro deve girare per forza con Windows, e il Green permette di essere gestito tramite App sul telefono. Opterò per un Mini pc.
Grazie mille per il chiarimento

**Reply by indomus** [reply]
Qui trovi tutte le guide passo passo: https://indomus.it/formazione/home-assistant-su-mini-pc-intel-nuc-le-varie-installazioni-le-cose-da-fare/

**Comment by Menhir83** [comment]
Ho capito, grazie mille! Mi farò l'ennesima rilettura perché davvero c'è
tanto da imparare

Il Mer 19 Giu 2024, 14:58 inDomus ***@***.***> ha scritto:
